sdk/servicebus/azure-servicebus/azure/servicebus/_servicebus_sender.py

```diff
@@ -221,16 +230,17 @@ def schedule_messages(self, messages, schedule_time_utc, **kwargs):
         timeout = kwargs.pop("timeout", None)
         if timeout is not None and timeout <= 0:
             raise ValueError("The timeout must be greater than 0.")
-        if isinstance(messages, Message):
-            request_body = self._build_schedule_request(schedule_time_utc, messages)
-        else:
-            request_body = self._build_schedule_request(schedule_time_utc, *messages)
-        return self._mgmt_request_response_with_retry(
-            REQUEST_RESPONSE_SCHEDULE_MESSAGE_OPERATION,
-            request_body,
-            mgmt_handlers.schedule_op,
-            timeout=timeout
-        )
+        with send_trace_context_manager(span_name=SPAN_NAME_SCHEDULE) as send_span:
+            if isinstance(messages, Message):
+                request_body = self._build_schedule_request(schedule_time_utc, send_span, messages)
+            else:
+                request_body = self._build_schedule_request(schedule_time_utc, send_span, *messages)
+            return self._mgmt_request_response_with_retry(
+                REQUEST_RESPONSE_SCHEDULE_MESSAGE_OPERATION,
+                request_body,
+                mgmt_handlers.schedule_op,
+                timeout=timeout
+            )
```

Comment:
I noticed in `send_messages` we would call `if send_span: self._add_span_request_attributes(send_span)`, we don't need one in `schedule_messages`?

Reply:
Good catch++  I'm inclined to think you're right for parity.